frontend/static/languages/_groups.json

```
@@ -171,7 +171,10 @@
   },
   {
     "name": "irish",
-    "languages": ["irish"]
+    "languages": [
+                  "irish",
+                  "irish_1k"
+    ]
```

```suggestion
    "languages": [
      "irish",
      "irish_1k"
    ]
```
Let's match the indent with the rest of the file.